package-lock.json

```diff
@@ -631,6 +631,25 @@
       "resolved": "https://registry.npmjs.org/@socket.io/component-emitter/-/component-emitter-3.1.0.tgz",
       "integrity": "sha512-+9jVqKhRSpsc591z5vX+X5Yyw+he/HCB4iQ/RYxw35CEPaY1gnsNE43nf9n9AaYjAQrTiI/mOwKUKdUs9vf7Xg=="
     },
+    "node_modules/@types/body-parser": {
```

Comment:
Does this affect dependencies that people take on if they depend on connect-web? If our examples are affecting the size of our libraries, we should move examples to another repo. I legitimately do not know how this ecosystem works, so wanted to ask

Reply:
Absolutely not. 

This lock file includes packages required for tests and benchmarks of the entire monorepo, but it's not used when you install Connect-Web. Connect-Web only needs @bufbuild/protobuf. This [package.json](https://github.com/bufbuild/connect-web/blob/main/packages/connect-web/package.json#L26)) is the authority for that.

Reply:
OK great, just making sure